setup.py

```diff
@@ -44,7 +44,9 @@ def read(fname):
     "matplotlib >= 3.3.0",
     "multiprocess",
     "networkx >= 2.5.0",
-    "pandas >= 1.4.0",
+    # newer pandas versions don't work with specified sqlalchemy versions, but upgrading
+    # sqlalchemy leads to new errors.. should be fixed at some point
+    "pandas < 2.2.0",
```

Comment:
I think this requirement should also be added to the yml files, for all windows users :)

Reply:
You're right, I added it.